dev/benchmarks/macrobenchmarks/test/util.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file probably also belongs to `test_driver` as this is unlikely to be used as a unit test. BTW, I guess we'll update `WidgetTester` to `WidgetController` in this file?

Reply:
I'm not expecting to use that demo very soon but if you think I should, maybe I'd better change this PR back to draft as that demo is far from use-able.

Reply:
And the reason I'm putting it in `test/` rather than test_driver is that E2E recommends so (https://pub.dev/packages/e2e#test-locations), and the test file (`cull_opacity_perf_e2e.dart`) should be able to run using `flutter test`.

Reply:
Changing from `WidgetTester` to `WidgetController` for `driverOps` and `setupOps` for now, but this specific case it's not used.